src/main/java/org/folio/list/services/export/CsvCreator.java

```diff
@@ -44,6 +46,9 @@ public class CsvCreator {
   private final EntityTypeClient entityTypeClient;
   private final FolioS3Client folioS3Client;
 
+  //Minimal s3 part size is 5 MB
+  private static final Long MINIMAL_PART_SIZE = 5242880L;
```

Comment:
Tiny thing, but this would probably be better as `MINIMAL_PART_SIZE_BYTES`, since `Long` doesn't capture the units